src/fcopymovedlg.lfm

```diff
@@ -10,6 +10,7 @@ object frmCopyDlg: TfrmCopyDlg
   ActiveControl = edtDst
   AutoSize = True
   BorderIcons = [biSystemMenu, biMaximize]
+  BorderStyle = bsSingle
```

Comment:
Is it a mandatory change? Can it be done only for Cocoa (via code)?

Reply:
1. it's not a mandatory requirement, not even for cocoa
2. it's just a convention, the dialog is fixed size like other dialogs in DC
3. CopyMoveDlg was originally set to be resizable, is there any special consideration?

Reply:
Many dialogs in DC are resizable. Only some dialogs has fixed size. User may want to resize CopyMoveDlg if full target path is very long.

Reply:
THX.
form BorderStyle setting restored.
check it please.